src/Properties/CoordinatesProperty.php

```diff
@@ -22,7 +22,7 @@ public function __construct(string $name, float $lat, float $lng)
 
     public function getValue(): string
     {
-        return "{$this->lat};{$this->lng}";
+        return json_encode($this->lat) . ';' . json_encode($this->lng);
```

Comment:
Using number_format would be more clear here.

Reply:
`number_format` requires us to specify a fixed number of decimal points, which is not useful.
`sprintf` cannot be used because it uses locale-specific decimal points.
The only options I see is using `json_encode` or `var_export(.., true)`. 

Would you like to have `var_export` instead of `json_encode`?